metadata_service/proxy/atlas_proxy.py

```diff
@@ -997,3 +997,37 @@ def get_resources_using_table(self, *,
                                   id: str,
                                   resource_type: ResourceType) -> Dict[str, List[DashboardSummary]]:
         return {}
+
+    def put_user(self, *, data: User) -> None:
+        pass
+
+    def post_users(self, *, data: List[User]) -> None:
+        pass
+
+    def put_app(self, *, data: Application) -> None:
+        pass
+
+    def post_apps(self, *, data: List[Application]) -> None:
+        pass
+
+    def put_table(self, *, table: Table) -> None:
+        pass
+
+    def post_tables(self, *, tables: List[Table]) -> None:
+        pass
+
+    def put_column(self, *, table_uri: str, column: Column) -> None:
+        pass
+
+    def put_programmatic_table_description(self, *, table_uri: str, description: ProgrammaticDescription) -> None:
+        pass
+
+    def put_programmatic_column_description(self, *, table_uri: str, column_name: str,
+                                            description: Description) -> None:
+        pass
+
+    def expire_columns(self, *, table_uri: str, column_names: List[str]) -> None:
```

Comment:
this one is square specific, I think!

Reply:
Ah, yes, you're right. I'll remove the expire_columns stuff for now.